client/js/overlays/states.js

```diff
@@ -63,7 +63,7 @@ async function addState(e, type, src, id) {
 
   let url = `/addState/${roomID}/${id}/${type}/${src && src.name && encodeURIComponent(src.name)}/`;
   waitingForStateCreation = id;
-  if(e && (e.target.parentNode == $('#addVariant') || e.target.classList.contains('update'))) {
+  if(e && (e.target.parentNode.parentNode == $('#addVariant') || e.target.classList.contains('update'))) {
```

Comment:
Might be better to change this to just look for #addVariant as an ancestor. Are you sure #addVariant is always the grandparenet element now?

Reply:
That's the structure of the html file from what I saw.